gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStatistic.java

```diff
@@ -39,40 +39,22 @@
 @InterfaceStability.Unstable
 public enum GhfsStatistic {
   /** HTTP request counters */
-  ACTION_HTTP_HEAD_REQUEST(
-      StoreStatisticNames.ACTION_HTTP_HEAD_REQUEST, "HEAD request.", TYPE_COUNTER),
   ACTION_HTTP_GET_REQUEST(
       StoreStatisticNames.ACTION_HTTP_GET_REQUEST, "GET request.", TYPE_COUNTER),
   ACTION_HTTP_PUT_REQUEST(
       StoreStatisticNames.ACTION_HTTP_PUT_REQUEST, "PUT request.", TYPE_COUNTER),
   ACTION_HTTP_PATCH_REQUEST(
       StoreStatisticNames.ACTION_HTTP_POST_REQUEST, "PATCH request.", TYPE_COUNTER),
-  ACTION_HTTP_POST_REQUEST(
-      StoreStatisticNames.ACTION_HTTP_PATCH_REQUEST, "POST request.", TYPE_COUNTER),
+
   ACTION_HTTP_DELETE_REQUEST(
       StoreStatisticNames.ACTION_HTTP_DELETE_REQUEST, "DELETE request.", TYPE_COUNTER),
-  ACTION_HTTP_HEAD_REQUEST_FAILURE(
-      "action_http_head_request_failures", "HEAD request failure.", TYPE_COUNTER),
-  ACTION_HTTP_GET_REQUEST_FAILURE(
-      "action_http_get_request_failures", "GET request failure.", TYPE_COUNTER),
-  ACTION_HTTP_PUT_REQUEST_FAILURE(
-      "action_http_put_request_failures", "PUT request failure.", TYPE_COUNTER),
-  ACTION_HTTP_PATCH_REQUEST_FAILURE(
-      "action_http_patch_request_failures", "PATCH request failure.", TYPE_COUNTER),
-  ACTION_HTTP_POST_REQUEST_FAILURE(
-      "action_http_post_request_failures", "POST request failure.", TYPE_COUNTER),
-  ACTION_HTTP_DELETE_REQUEST_FAILURE(
-      "action_http_delete_request_failures", "DELETE request failure.", TYPE_COUNTER),
 
   /** FileSystem-level statistics */
   DIRECTORIES_CREATED(
       "directories_created",
       "Total number of directories created through the object store.",
       TYPE_COUNTER),
-  DIRECTORIES_DELETED(
```

Comment:
why removing this?

Reply:
Directories Deleted isn't used in master, only functions were defined but never actually incremented.